src/Init/Data/Char/Tables.lean

```diff
@@ -0,0 +1,45 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jean-Baptiste Tristan
+-/
+import Init.Data.Char.UnicodeSkipList
+
```

Comment:
We should put a comment here referencing how the data is generated, as well as to `Char.unicodeVersion`